lib/cisco_node_utils/bgp_af.rb

```diff
@@ -59,7 +59,10 @@ def self.afs
     end
 
     def create
-      Feature.bgp_enable if platform == :nexus
+      if platform == :nexus
+        Feature.bgp_enable
+        Feature.nv_overlay_evpn_enable if @safi[/evpn/]
```

Comment:
With this conditional enable, do we still need this?

https://github.com/cisco/cisco-network-node-utils/blob/develop/lib/cisco_node_utils/bgp_af.rb#L157